src/malawi/mwi_seasonalrainfallforecasts.py

```diff
@@ -1,3 +1,5 @@
+# TODO: this scripts needs serious refactoring
```

Comment:
Can add these kinds of comments to an issue so that we can keep better track or refactoring priorities?

Reply:
Yes good one. I will make an issue of converting large todo's to issues
And for this file specifically, it is actually never used so deteleted it (easy fix :p )